server/nuxeo-launcher/src/main/resources/log4j2.xml

```diff
@@ -1,9 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<Configuration>
+<Configuration shutdownHook="disable">
+  <!--
+    Nuxeo Launcher log4j2 configuration file (aka nuxeoctl).
+    We define two console appenders, one for stdout and one for stderr.
+    When using nuxeo in console mode and an exit signal is received (ctrl+c), console output becomes the stderr output,
+    thus the need of two appenders.
```

Comment:
Not sure to understand does this mean that "nuxeoctl console" use stdout and stderr except
when SIGINT is received? 
what do you mean by "console output" is it "nuxeoctl console" output or appender console output?
Also, Ctrl-C is an interrupt signal not exit signal

Reply:
I will rework this section, basically we have for:
- nuxeoctl start: stdin/stderr/stdout for nuxeoctl launcher
- nuxeoctl console: stdin for nuxeoctl launcher, stdout/stderr for launcher + nuxeo and only stderr when SIGINT is received.